src/App.js

```diff
@@ -78,10 +80,9 @@ class App extends Component {
         </div>
         <Load load={this.state.load} />
         <Loader isLoading={this.state.isLoading} />
-        <div className="container">
-          <div className="row">
-            <div className="col-md-4"><Main_Video video={this.state.URL} /></div>
-          </div>
+        <div className="GridSystem">
+          <div><Main_Video video={this.state.URL} /></div>
```

Comment:
Don't mix the child tags in the same line. Separate into next lines.
Do it for the rest of the files if any.